apm-sniffer/config/agent.config

```diff
@@ -320,3 +320,17 @@ plugin.nettyhttp.supported_content_types_prefix=${SW_PLUGIN_NETTYHTTP_SUPPORTED_
 plugin.rocketmqclient.collect_message_keys=${SW_PLUGIN_ROCKETMQCLIENT_COLLECT_MESSAGE_KEYS:false}
 # If set to true, the tags of messages would be collected by the plugin for RocketMQ Java client.
 plugin.rocketmqclient.collect_message_tags=${SW_PLUGIN_ROCKETMQCLIENT_COLLECT_MESSAGE_TAGS:false}
+# Define the max length of collected HTTP parameters. The default value(=0) means not collecting.
+plugin.solon.http_params_length_threshold=${SW_PLUGIN_SOLON_HTTP_PARAMS_LENGTH_THRESHOLD:0}
+# Define the max length of collected HTTP header. The default value(=0) means not collecting.
+plugin.solon.http_headers_length_threshold=${SW_PLUGIN_SOLON_HTTP_HEADERS_LENGTH_THRESHOLD:0}
+# It controls what header data should be collected, values must be in lower case, if empty, collect all. default is empty.
+plugin.solon.include_http_headers=${SW_PLUGIN_SOLON_INCLUDE_HTTP_HEADERS:}
```

Comment:
Again, you should not collect all headers. All collected things should be very specific.

Header usually carries auth token, this is auto collecting is dangerous and could cause sensitive data leak.